Receptai.client/components/RecipeContainerList/RecipeContainerList.vue

```diff
@@ -76,7 +77,7 @@ try {
       v-for="item in recipeList"
       :imageLink="item.previewImage"
       :name="item.name"
-      :raiting="5"
+      :raiting="item.averageRating"
```

Comment:
Reiktu kažkada šitą prop pervadint iš raiting į rating [Receptai.client/components/RecipeContainerComponent/RecipeContainerComponent.vue](https://github.com/Tasty-Bytes-Team/Receptai/pull/22/files#diff-923bf39afe70a60e02e1a6e759330b9b7231cbb686058377a6a48150d38d960a)